data_subscriptions/api/resources/subscription.py

```diff
@@ -105,6 +110,26 @@ def __init__(self):
         self.NEW_DATASETS = "NEW_DATASETS"
         self.DELETE = "DELETE"
         self.POST = "POST"
+        self.PUT = "PUT"
+
+    def put(self):
+        response = {}
+        data = request.get_json(force=True)
+        keys = ["email", "user_id", "username", "kind"]
```

Comment:
`user_id` is the unique id for the user in kratos, `username` is the name of the user, so for instance you could have someone with two accounts, one in github and one in hotmail, they would both have the same usernames both different ids, as to why we store this information, i believe is because there is a `stat` route that seems to generate a .csv with a subscription report. altough i wasnt able to test it on my local environemnt

Reply:
ok, got it @luccasmmg thanks for clarifying, good to know 👍🏼